pkg/virt-launcher/virtwrap/cmd-server/server.go

```diff
@@ -237,7 +237,7 @@ func (l *Launcher) GetDomain(ctx context.Context, request *cmdv1.EmptyRequest) (
 		return response, nil
 	}
 
-	if len(list) >= 0 {
+	if len(list) > 0 {
```

Comment:
IIRC I fixed the same in GetDomainStats few lines below recently... too bad I missed that we probably copy/pasted the error to another place. Nice catch!

Reply:
Thanks to logs:))